README.md
# Next.js 13 Full Course 2023 | Build and Deploy a Full Stack App Using the Official React Framework
![AI Prompt Sharing](https://i.ibb.co/9pQNZZy/Thumbnail-27.png)

### [🌟 Become a top 1% Next.js 13 developer in only one course](https://jsmastery.pro/next13)
### [🚀 Land your dream programming job in 6 months](https://jsmastery.pro/masterclass)

## Introduction
Next.js recently became the official React framework as outlined in React docs. In this course, you'll learn the most important Next.js concepts and how they fit into the React ecosystem. Finally, you'll put your skills to the test by building a modern full-stack Next 13 application.
 
In this video, you'll learn:
- Next.js 13 App Folder Structure
- Next.js 13 Client Components vs Server Components
- Next.js 13 File-based Routing (including dynamic and nested routes)
- Next.js 13 page, layout, loading, and error Special Files
- Next.js 13 Serverless Route Handlers (Next API, Full Stack Apps)
- Next.js 13 Metadata and Search Engine Optimization (SEO)
- Three ways to fetch data in Next.js:
   - Server Side Rendering (SSR),
   - Static Site Generation (SSG)
   - Incremental Static Generation (ISR)

## Want to land your dream programming job in 3 - 6 months?
⭐ JSM Masterclass Experience - https://jsmastery.pro/masterclass
Become a Software Engineer. Guaranteed.

## Testing
I'm testing to lean PR submission


primereact-sass-theme-9.6.2/CHANGELOG.md
# Changelog
## [9.6.2](https://github.com/primefaces/primereact-sass-theme/tree/9.6.2) (2023-08-17)

[Full Changelog](https://github.com/primefaces/primereact-sass-theme/compare/9.6.1...9.6.2)

## [9.6.1](https://github.com/primefaces/primereact-sass-theme/tree/9.6.1) (2023-08-01)

[Full Changelog](https://github.com/primefaces/primereact-sass-theme/compare/9.6.0...9.6.1)

## [9.6.0](https://github.com/primefaces/primereact-sass-theme/tree/9.6.0)  (2023-06-27)

[Full Changelog](https://github.com/primefaces/primereact-sass-theme/compare/9.5.0...9.6.0) 

**Fixed bugs:**

- SplitButton: menu icon not visible in large size [\#18](https://github.com/primefaces/primereact-sass-theme/issues/18)

## [9.5.0](https://github.com/primefaces/primereact-sass-theme/tree/9.5.0) (2023-05-24)

[Full Changelog](https://github.com/primefaces/primereact-sass-theme/compare/9.4.0...9.5.0)

**Implemented Enhancements:**

- Messages: Inconsistency [\#16](https://github.com/primefaces/primereact-sass-theme/issues/16)

**Fixed bugs:**

- BreadCrumb: SVG location of the DOM [\#15](https://github.com/primefaces/primereact-sass-theme/issues/15)

## [9.4.0](https://github.com/primefaces/primereact-sass-theme/tree/9.4.0) (2023-05-12)

[Full Changelog](https://github.com/primefaces/primereact-sass-theme/compare/9.3.1...9.4.0)

**Fixed bugs:**

- Enhancement: Dialog Component Styling Update [\#4](https://github.com/primefaces/primereact-sass-theme/issues/4)

## [9.3.1](https://github.com/primefaces/primereact-sass-theme/tree/9.3.1) (2023-05-03)

- No changes, provides semantic versioning compatibility for PrimeReact 9.3.1

## [9.3.0](https://github.com/primefaces/primereact-sass-theme/tree/9.3.0) (2023-04-19)

[Full Changelog](https://github.com/primefaces/primereact-sass-theme/compare/9.2.3...9.3.0)

**Implemented New Features and Enhancements:**

- Refactor the icons that getting font-size variable [\#2](https://github.com/primefaces/primereact/issues/2)
- SVG: icon width & height fixes [\#1](https://github.com/primefaces/primereact/issues/1)

## [9.2.3](https://github.com/primefaces/primereact-sass-theme/tree/9.2.3) (2023-04-01)

- No changes, provides semantic versioning compatibility for PrimeReact 9.2.3

## [9.2.2](https://github.com/primefaces/primereact-sass-theme/tree/9.2.2) (2023-03-28)

- No changes, provides semantic versioning compatibility for PrimeReact 9.2.2

## [9.2.1](https://github.com/primefaces/primereact-sass-theme/tree/9.2.1) (2023-03-15)

- No changes, provides semantic versioning compatibility for PrimeReact 9.2.1

## [9.2.0](https://github.com/primefaces/primereact-sass-theme/tree/9.2.0) (2023-03-13)

- Initial release

app/editor/data.js
import React from "react";
const columns = [
    { name: "ID", uid: "id", sortable: true },
    { name: "NAME", uid: "name", sortable: true },
    { name: "AGE", uid: "age", sortable: true },
    { name: "ROLE", uid: "role", sortable: true },
    { name: "TEAM", uid: "team" },
    { name: "EMAIL", uid: "email" },
    { name: "STATUS", uid: "status", sortable: true },
    { name: "ACTIONS", uid: "actions" },
];

const statusOptions = [
    { name: "Active", uid: "active" },
    { name: "Paused", uid: "paused" },
    { name: "Vacation", uid: "vacation" },
];

const users = [
    {
        id: 1,
        name: "Tony Reichert",
        role: "CEO",
        team: "Management",
        status: "active",
        age: "29",
        avatar: "https://i.pravatar.cc/150?u=a042581f4e29026024d",
        email: "tony.reichert@example.com",
    },
    {
        id: 2,
        name: "Zoey Lang",
        role: "Tech Lead",
        team: "Development",
        status: "paused",
        age: "25",
        avatar: "https://i.pravatar.cc/150?u=a042581f4e29026704d",
        email: "zoey.lang@example.com",
    },
    {
        id: 3,
        name: "Jane Fisher",
        role: "Sr. Dev",
        team: "Development",
        status: "active",
        age: "22",
        avatar: "https://i.pravatar.cc/150?u=a04258114e29026702d",
        email: "jane.fisher@example.com",
    },
    {
        id: 4,
        name: "William Howard",
        role: "C.M.",
        team: "Marketing",
        status: "vacation",
        age: "28",
        avatar: "https://i.pravatar.cc/150?u=a048581f4e29026701d",
        email: "william.howard@example.com",
    },
    {
        id: 5,
        name: "Kristen Copper",
        role: "S. Manager",
        team: "Sales",
        status: "active",
        age: "24",
        avatar: "https://i.pravatar.cc/150?u=a092581d4ef9026700d",
        email: "kristen.cooper@example.com",
    },
    {
        key: 6,
        name: "Brian Kim",
        role: "P. Manager",
        team: "Management",
        age: "29",
        avatar: "https://i.pravatar.cc/150?u=a042581f4e29026024d",
        email: "brian.kim@example.com",
        status: "Active",
    },
    {
        id: 7,
        name: "Michael Hunt",
        role: "Designer",
        team: "Design",
        status: "paused",
        age: "27",
        avatar: "https://i.pravatar.cc/150?u=a042581f4e29027007d",
        email: "michael.hunt@example.com",
    },
    {
        id: 8,
        name: "Samantha Brooks",
        role: "HR Manager",
        team: "HR",
        status: "active",
        age: "31",
        avatar: "https://i.pravatar.cc/150?u=a042581f4e27027008d",
        email: "samantha.brooks@example.com",
    },
    {
        id: 9,
        name: "Frank Harrison",
        role: "F. Manager",
        team: "Finance",
        status: "vacation",
        age: "33",
        avatar: "https://i.pravatar.cc/150?img=4",
        email: "frank.harrison@example.com",
    },
    {
        id: 10,
        name: "Emma Adams",
        role: "Ops Manager",
        team: "Operations",
        status: "active",
        age: "35",
        avatar: "https://i.pravatar.cc/150?img=5",
        email: "emma.adams@example.com",
    },
    {
        id: 11,
        name: "Brandon Stevens",
        role: "Jr. Dev",
        team: "Development",
        status: "active",
        age: "22",
        avatar: "https://i.pravatar.cc/150?img=8",
        email: "brandon.stevens@example.com",
    },
    {
        id: 12,
        name: "Megan Richards",
        role: "P. Manager",
        team: "Product",
        status: "paused",
        age: "28",
        avatar: "https://i.pravatar.cc/150?img=10",
        email: "megan.richards@example.com",
    },
    {
        id: 13,
        name: "Oliver Scott",
        role: "S. Manager",
        team: "Security",
        status: "active",
        age: "37",
        avatar: "https://i.pravatar.cc/150?img=12",
        email: "oliver.scott@example.com",
    },
    {
        id: 14,
        name: "Grace Allen",
        role: "M. Specialist",
        team: "Marketing",
        status: "active",
        age: "30",
        avatar: "https://i.pravatar.cc/150?img=16",
        email: "grace.allen@example.com",
    },
    {
        id: 15,
        name: "Noah Carter",
        role: "IT Specialist",
        team: "I. Technology",
        status: "paused",
        age: "31",
        avatar: "https://i.pravatar.cc/150?img=15",
        email: "noah.carter@example.com",
    },
    {
        id: 16,
        name: "Ava Perez",
        role: "Manager",
        team: "Sales",
        status: "active",
        age: "29",
        avatar: "https://i.pravatar.cc/150?img=20",
        email: "ava.perez@example.com",
    },
    {
        id: 17,
        name: "Liam Johnson",
        role: "Data Analyst",
        team: "Analysis",
        status: "active",
        age: "28",
        avatar: "https://i.pravatar.cc/150?img=33",
        email: "liam.johnson@example.com",
    },
    {
        id: 18,
        name: "Sophia Taylor",
        role: "QA Analyst",
        team: "Testing",
        status: "active",
        age: "27",
        avatar: "https://i.pravatar.cc/150?img=29",
        email: "sophia.taylor@example.com",
    },
    {
        id: 19,
        name: "Lucas Harris",
        role: "Administrator",
        team: "Information Technology",
        status: "paused",
        age: "32",
        avatar: "https://i.pravatar.cc/150?img=50",
        email: "lucas.harris@example.com",
    },
    {
        id: 20,
        name: "Mia Robinson",
        role: "Coordinator",
        team: "Operations",
        status: "active",
        age: "26",
        avatar: "https://i.pravatar.cc/150?img=45",
        email: "mia.robinson@example.com",
    },
];

export { columns, users, statusOptions };
